### PR TITLE
Add base styles

### PR DESCRIPTION
**Pull request by Ngaremaina**
# Description

Creating an empty file that is called content development page

Issue #12

Fixes # (29) 
- *Example*: Fixes #10, #20 NOTE: GitHub issue numbers should be prefixed with **#** to aid with issue tracking.

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] The file is executed in the browser
- [x] The file can display the elements in the file

**Test Configuration**: (optional)

- Firmware version:
- Hardware:
- Toolchain:
- SDK:

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ Here is what you need to be able to run the App.
 
 ### Setup
 
-1. Clone the repo into a public GitHub repository (fork <https://github.com/italanta/elewa-website-html/fork>).
+1. Fork then clone repo from your public GitHub repository (fork <https://github.com/italanta/elewa-website-html/fork>).
 
    ```sh
    git clone https://github.com/{yourusername}/elewa-website-html.git
```
Comment (Beulah-Matt) [inline README.md:46]: Check file references. These changes should not be here.

File: src/features/pages/consultancy-page.html [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Consultancy Page</title>
+</head>
+<body>
+    <div>Consultancy Page</div>
+    
+</body>
+</html>
```
Comment (Beulah-Matt) [inline src/features/pages/consultancy-page.html:12]: Your PR links issue 12, but you have here changes for another issue. Always make sure you work on one issue at a time.

**Review by Beulah-Matt — COMMENTED** [review]
This PR is based of off a different branch, instead of the default *main* branch (Main). Check on this next mine. 
Also, it is important that you do not mix issues.

**Comment by Beulah-Matt** [comment]
Closing this due to conflicting workflows.

**Comment by Ngaremaina** [comment]
Hello Mr. Mathenge,

My apologies for the confusion in resolving the issues. I would like to
confirm on something, are we supposed to do the issues on the main branch
or should we do them on different branches?

Thank you for your time.

**Comment by Beulah-Matt** [comment]
Hi @Ngaremaina , 

What I meant is, you should compare your changes to main. 
What you have is 
image

But ideally you should have 
image

Your comparison was wrong.

PS: I am miss Mathenge, but Beulah is just fine :-)

**Comment by Ngaremaina** [comment]
My apologies for the inconveniences. Thank you very much for the
clarification. I will improve on that Beulah.

My sincere gratitude.

**Pull request closed without merging**